docs/testing-teal.md

```diff
@@ -131,15 +236,19 @@ Currently, `runtime` supports:
   + `delete` application
   + `closeout` from an application
   + `clearState` of application
+  + `create` an asset
+  + `opt-in` to asset
+  + `transfer` an asset
 
 - Full transaction processing for type `payment`, `application call`
```

Comment:
I think now we support full transaction processing for all types. Can we remove/modify this?

Reply:
no, we don't for all. we will update it after asset support (revoke, freeze, etc) gets merged.

Reply:
ah, yes. It's not merged. We can update there.